$MirOS: ports/devel/autoconf/2.61/patches/patch-doc_autoconf_texi,v 1.1 2007/05/08 20:38:25 tg Exp $
--- doc/autoconf.texi.orig	2008-09-09 14:56:54.000000000 +0200
+++ doc/autoconf.texi	2009-07-14 21:52:26.000000000 +0200
@@ -205,27 +205,32 @@ freedom.''
 
 @dircategory Software development
 @direntry
-* Autoconf: (autoconf).         Create source code configuration scripts.
+* Autoconf-2.63: (autoconf-2.63).
+                                Create source code configuration scripts.
 @end direntry
 
 @dircategory Individual utilities
 @direntry
-* autoscan: (autoconf)autoscan Invocation.
+* autoscan-2.63: (autoconf-2.63)autoscan Invocation.
                                 Semi-automatic @file{configure.ac} writing
-* ifnames: (autoconf)ifnames Invocation.        Listing conditionals in source.
-* autoconf-invocation: (autoconf)autoconf Invocation.
+* ifnames-2.63: (autoconf-2.63)ifnames Invocation.
+                                Listing conditionals in source.
+* autoconf-2.63-invocation: (autoconf-2.63)autoconf Invocation.
                                 How to create configuration scripts
-* autoreconf: (autoconf)autoreconf Invocation.
+* autoreconf-2.63: (autoconf-2.63)autoreconf Invocation.
                                 Remaking multiple @command{configure} scripts
-* autoheader: (autoconf)autoheader Invocation.
+* autoheader-2.63: (autoconf-2.63)autoheader Invocation.
                                 How to create configuration templates
-* autom4te: (autoconf)autom4te Invocation.
+* autom4te-2.63: (autoconf-2.63)autom4te Invocation.
                                 The Autoconf executables backbone
-* configure: (autoconf)configure Invocation.    Configuring a package.
-* autoupdate: (autoconf)autoupdate Invocation.
+* configure-2.63: (autoconf-2.63)configure Invocation.
+                                Configuring a package.
+* autoupdate-2.63: (autoconf-2.63)autoupdate Invocation.
                                 Automatic update of @file{configure.ac}
-* config.status: (autoconf)config.status Invocation. Recreating configurations.
-* testsuite: (autoconf)testsuite Invocation.    Running an Autotest test suite.
+* config.status-2.63: (autoconf-2.63)config.status Invocation.
+                                Recreating configurations.
+* testsuite-2.63: (autoconf-2.63)testsuite Invocation.
+                                Running an Autotest test suite.
 @end direntry
 
 @titlepage
